.travis.yml

```diff
@@ -15,9 +15,5 @@ install:
   - npm i -g npm@latest
   - npm i
 
-script:
-  - npm run lint
-  - npm run test
-
```

Comment:
i feel this might bite us later, what if travis silently stops linting ? or changes the test target ?